app/battle/contest.py

```diff
@@ -0,0 +1,55 @@
+from ..knight.knight import Knight
+
+
+class Contest:
+    def __init__(self) -> None:
+        print("Welcome to contest!")
+
+    @staticmethod
+    def create_knight_instances(ritters: dict) -> list:
+        knight_instances = list()
+
+        for ritter in ritters:
+            knight_instances.append(Knight(
+                ritters[ritter]["name"],
+                ritters[ritter]["power"],
+                ritters[ritter]["hp"],
+                ritters[ritter]["armour"],
+                ritters[ritter]["weapon"],
+                ritters[ritter]["potion"]
+            ))
+
+        return knight_instances
```

Comment:
you can save knight_instances as Contest attribute and use it in other functions